Tables/Tab81751.Result.al

```diff
@@ -0,0 +1,78 @@
+/// <summary>
+/// Table to store the result of the operations
+/// </summary>
+table 81754 Result
+{
+    Caption = 'Result';
+    DataClassification = SystemMetadata;
+
+    fields
+    {
+        field(1; "Entry No."; Integer)
+        {
+            Caption = 'Entry No.';
+            AutoIncrement = true;
+        }
+        field(2; Operation; Text[100])
+        {
+            Caption = 'Operation';
+        }
+        field(3; "Duration"; Duration)
+        {
+            Caption = 'Duration';
+        }
+        field(4; Comment; Text[250])
+        {
+            Caption = 'Comment';
+        }
+        field(5; "No of Iterations"; Integer)
+        {
+            Caption = 'No of Iterations';
+            Editable = false;
+        }
+        field(6; "Duration Int"; Integer)
+        {
+            Caption = 'Duration';
+        }
+        field(7; "Sql Rows Read"; BigInteger)
+        {
+            Caption = 'Sql Rows Read';
+            Editable = false;
+        }
+        field(8; "Sql Statements Executed"; BigInteger)
+        {
+            Caption = 'Sql Statements Executed';
+            Editable = false;
+        }
+
+    }
+    keys
+    {
+        key(PK; "Entry No.")
+        {
+            Clustered = true;
+        }
+    }
+
+    // procedure Log(Operation: Text[100]; Duration: Duration; NoOfIterations: Integer; Comment: Text[250])
```

Comment:
@NAVFreak forgotten code

Reply:
Previous function that got outdated. Didn't spend time on cleaning